indico/modules/events/editing/client/js/components/Timeline/judgment/TagInput.jsx

```diff
@@ -0,0 +1,78 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Dropdown} from 'semantic-ui-react';
+import {FinalField} from 'indico/react/forms/fields';
+import {Translate} from 'indico/react/i18n';
+
+import './TagInput.module.scss';
+
+/**
+ * A dropdown which allows tags to be searched and added
+ */
+function TagInput({onChange, value: defaultValue, placeholder, options}) {
+  return (
+    <Dropdown
+      placeholder={placeholder}
+      styleName="tag-input"
+      upward={false}
+      fluid
+      multiple
+      search
+      selection
+      defaultValue={defaultValue}
+      options={options.map(({id, code, title, ...rest}) => ({
+        value: id,
+        text: `${code}: ${title}`,
```

Comment:
what about adding a `verbose_title` to the schema for this? otherwise we'll probably have inconsistent styles (`: ` vs ` - `) at some point

Reply:
Good idea.